cli/smartcontract/smart_contract.go

```diff
@@ -524,7 +518,7 @@ func invokeInternal(ctx *cli.Context, signAndPush bool) error {
 	if !args.Present() {
 		return cli.NewExitError(errNoScriptHash, 1)
 	}
-	script, err := util.Uint160DecodeStringLE(args[0])
+	script, err := flags.ParseAddress(args[0])
```

Comment:
Let's then use it as an address flag in `cli.Command`'s description instead of using a simple string flag.

I.g. like we have for `wallet register` command:
```
Flags: append([]cli.Flag{
				walletPathFlag,
				gasFlag,
				flags.AddressFlag{
					Name:  "address, a",
					Usage: "Address to register",
				},
			}
```

Then we can just do
```
addrFlag := ctx.Generic("address").(*flags.Address)
addr := addrFlag.Uint160()
```

without errors handling.

Reply:
I think with `invoke*` positional argument is more convenient.